src/cloud-providers/docker/provider.go

```diff
@@ -72,11 +72,11 @@ func (p *dockerProvider) CreateInstance(ctx context.Context, podName, sandboxID
 
 	// Create volume binding for the container
 
-	// mount userdata to DockerUserDataUrl=/peerpods/userdata.json
+	// mount userdata to /peerpods/userdata.json
 	// This file will be read by process-user-data and daemon.json will be written to
 	// /run/peerpods/daemon.json at runtime
 	volumeBinding := []string{
-		fmt.Sprintf("%s:%s", instanceUserdataFile, DockerUserDataUrl),
+		fmt.Sprintf("%s:%s", instanceUserdataFile, "/peerpods/userdata.json"),
```

Comment:
I guess we don't want to pull in the `DockerUserDataPath` from paths and we want have the cloud-providers->CAA cyclic dependency. Maybe a comment here and in paths.go to say they have to stay in sync is the least bad approach then?